utils/chunk-tests.js

```diff
@@ -36,13 +70,12 @@ function getRunnerOptions(scriptName, packageName) {
       packageOptionsOverrides[packageName]
     );
   }
-  return (
-    runnerOptions || {
-      min: 1,
-      max: 1,
-      runners: ['ubuntu-latest'],
-    }
-  );
+  if (!runnerOptions) {
```

Comment:
I couldn't understand a scenario where we'd want a fallback value for this so got rid of it.